yorkie/src/main/kotlin/dev/yorkie/document/crdt/CrdtObject.kt

```diff
@@ -0,0 +1,128 @@
+package dev.yorkie.document.crdt
+
+import dev.yorkie.document.time.TimeTicket
+
+/**
+ * [CrdtObject] represents object datatype, but unlike regular JSON, it has time
+ * tickets which is created by logical clock.
+ */
+internal class CrdtObject private constructor(
+    createdAt: TimeTicket,
+    private val memberNodes: RhtPQMap<CrdtElement>,
+) : CrdtContainer(createdAt) {
+    val Rht
+        get() = memberNodes
```

Comment:
maybe it's better to make `memberNodes` public, but not sure

Reply:
I think it would be better to keep `memberNodes` private but provide a property returning `Iterable<RhtPQMap.RhtPQMapNode<T>>` type. 
this way we can hide mutability.